source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,102 @@ the Query API:
+     - :manual:`$dateToString </reference/operator/aggregation/dateToString/>`
+
+   * - `millisecondsAsDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#millisecondsAsDate()>`__
+     - :manual:`$toDate </reference/operator/aggregation/toDate/>`     
+
+   * - `parseDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseDate()>`__
+     - :manual:`$dateFromString </reference/operator/aggregation/dateFromString/>`
+
+   * - `dayOfWeek() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfWeek(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfWeek </reference/operator/aggregation/dayOfWeek/>`
+
+   * - `dayOfMonth() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfMonth(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfMonth </reference/operator/aggregation/dayOfMonth/>`
+
+   * - `dayOfYear() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#dayOfYear(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$dayOfYear </reference/operator/aggregation/dayOfYear/>`
+
+   * - `year() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#year(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$year </reference/operator/aggregation/year/>`
+
+   * - `month() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#month(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$month </reference/operator/aggregation/month/>`
+   
+   * - `week() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#week(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$week </reference/operator/aggregation/week/>`
+
+   * - `hour() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#hour(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$hour </reference/operator/aggregation/hour/>`
+
+   * - `minute() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#minute(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$minute </reference/operator/aggregation/minute/>`
+
+   * - `second() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#second(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$second </reference/operator/aggregation/second/>`
+
+   * - `millisecond() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlDate.html#millisecond(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$millisecond </reference/operator/aggregation/millisecond/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you have some UTC time series data and need to
```

Comment:
[grammar]
Since "UTC time series" is being used as a compound adjective instead of as a noun, it should be hyphenated. I pulled out the UTC because having the double hypen looked odd to me.

```suggestion
Suppose you have some time-series data in UTC and need to
```

Reply:
Alternative, you could reword to change "time series" into a noun.

```suggestion
Suppose you have a time series in UTC and need to
```

Reply:
added the full name to follow the style guide